README.md

```diff
@@ -655,38 +746,63 @@
     });
     ```
 
-  - Place 1 space before anonymous function parenthesis.
+  - Place 1 space before the opening parenthesis in control statements (`if`, `while` etc.). Place no space before the argument list in function calls and declarations.
 
     ```javascript
-	// bad
```

Comment:
Yeah, I totally get that. Like I said, if I received a pull request arguing for this as being our new accepted standard, I probably wouldn't vote against it.